docs/src/notebooks/9_max_min_driving_force_analysis.jl

```diff
@@ -97,26 +97,30 @@ reaction_standard_gibbs_free_energies = Dict( # kJ/mol
     "GLUt2r" => -3.49,
     "PPS" => -6.05,
     "FUM" => -3.42,
-)
-
-# In general you cannot be certain that all fluxes will be positive. This poses problems
-# for systematically enforcing that ΔᵣG ≤ 0 as done in the original formulation. In COBREXA
-# ΔᵣG ⋅ vᵢ ≤ 0 is instead enforced, where vᵢ is the flux of reaction i. By default all fluxes
-# are assumed to be positive, but by supplying thermodynamically consistent flux solution
-# it is possible to drop this implicit assumption and makes it easier to directly incorporate
-# the max min driving force into non-customized models.
+);
 
-flux_solution = flux_balance_analysis_dict(
+# In general you cannot be certain that all fluxes will be positive for a given flux
+# solution. This poses problems for systematically enforcing that ΔᵣG ≤ 0 for each reaction,
+# because it implicitly assumes that all fluxes are positive, as done in the original
+# formulation of MMDF. In COBREXA we instead enforce ΔᵣG ⋅ vᵢ ≤ 0, where vᵢ is the flux of
```

Comment:
"in `min_max_driving_force`" instead of COBREXA -- there might eventually be a different function that does it the other way.